chain/chain/src/update_shard.rs

```diff
@@ -61,37 +61,37 @@ pub enum ShardUpdateResult {
 pub enum ShardBlockUpdateResult {
     NewChunk(NewChunkResult),
     OldChunk(OldChunkResult),
-    StateSplit(StateSplitResult),
+    Resharding(ReshardingResult),
 }
 
-/// State roots of split shards which are ready.
-type SplitStateRoots = HashMap<ShardUId, StateRoot>;
+/// State roots of children shards which are ready.
+type ReshardingStateRoots = HashMap<ShardUId, StateRoot>;
```

Comment:
`ReshardedStateRoots`? Or actually, I think it's better to keep all names aligned with the `resharding` substring.

Reply:
Yeah I think greppability is an important factor and "resharding" is the string ;)